packages/astro/src/integrations/index.ts

```diff
@@ -11,13 +12,35 @@ import type { SerializedSSRManifest } from '../core/app/types';
 import type { PageBuildData } from '../core/build/types';
 import { mergeConfig } from '../core/config.js';
 import type { ViteConfigWithSSR } from '../core/create-vite.js';
+import { info, LogOptions } from '../core/logger/core.js';
+
+async function withTakingALongTimeMsg<T>({
```

Comment:
great function name :)

Reply:
😆 Played with some short names and thought... why not name it what is is?